deploy/demo/docker-compose.yml

```diff
@@ -212,7 +212,7 @@ services:
       - ./logs/keycloak:/opt/jboss/keycloak/standalone/log/:delegated
 
   kong:
-    image: kong:2.3
+    image: kong:2.4
```

Comment:
Is there any reason why we change from 2.3 to 2.4 (is it required by Jaeger?) ?

Reply:
Yes, Zipkin plugin included on 2.3 doesn't have Jaeger support